exercises/isogram/isogram.tc

```diff
@@ -0,0 +1,49 @@
+log←'./isogram.tc.log'
+⍎ ')COPY ',(⎕FIO 30),'/../../test.apl'
+
+test∆copy_relative 'isogram.apl'
+test∆clear_log log
+
+⍝ check test version
+  1⎕CR	(version)
+'1.2.0'
+
+⍝ ∇isIsogram
```

Comment:
I've been using underscore_case for the rest of the code, which appears to be a weak preference in modern APL (makes sense, since APL didn't originally support more than one case of letters).